docs/installation.md

```diff
@@ -1,3 +1,7 @@
 # Installation
 
-To Be Defined
\ No newline at end of file
+To install Shift on your machine simply download the .zip file build for the Python version of your liking from the [website](https://https://www.inbibo.co.uk/shift). Shift ditributions come with their own Python interpreter with all the required third-party libraries already packaged.
```

Comment:
Not a comment on what is written but a general question. Why are we shipping Shift in a zip file with a .bat?
It woule be more professional to make an installer with an `exe` to launch Shift. This way we could also configure file association (when a user double clicks an .sft file in their file browser it would open shift with that workflow)

Reply:
That is a whole process we should go through to make it happen. I did some research but python ways of packaging everything inside an exe file don't play nice with pyarmor. I managed to do it but the way is very hacky and it is definitely not good for a release atm.